sycl/doc/syclcompat/README.md

```diff
@@ -1457,6 +1457,15 @@ static void invoke_kernel_function(kernel_function &function,
 
 ### Math Functions
 
+The `funnelshift_*` APIs take two unsigned integer arguments (`low` and `high`),
+concatenate them, bit-shift the 64-bit result by `shift` bits and return either
+the most or least significant 32-bits. The `_l*` variants shift left and return
```

Comment:
First sentence very wordy and hard to follow. There is likely a much easier way to explain this operation. It should probably also be 2 sentences

Reply:
Good feedback, thanks @AidanBeltonS. I've addressed this now.